### PR TITLE
fix(deploy/w3c): allow publishing multiple times a day

### PR DESCRIPTION
**Pull request by sidvishnoi**
Fixes https://github.com/w3c/spec-prod/issues/41

### CODE REVIEW

File: src/prepare-build.ts [hunk 1]
```diff
@@ -291,8 +293,19 @@ async function getPreviousVersionInfo(shortName: string, publishDate: string) {
 
 		const thisDate = thisURI.match(/[1-2][0-9]{7}/)![0];
 		const targetPublishDate = publishDate.replace(/\-/g, "");
-		const currentURI =
-			thisDate === targetPublishDate && previousURI ? previousURI : thisURI;
+		const currentURI = (() => {
+			if (thisDate === targetPublishDate || !previousURI) {
+				console.log(
+					`[INFO] Document was published on same date (${publishDate}) earlier.`,
```
Comment (sidvishnoi) [inline src/prepare-build.ts:299]: `publishDate ~ targetPublishDate == thisDate`

**Review by sideshowbarker — APPROVED** [review]

**Comment by sidvishnoi** [comment]
I'm not fully convinced this is the right fix. I'll go through https://github.com/w3c/specberus/blob/3a69061a8a70943201577d03df1d77d2a09fd63e/lib/rules/headers/dl.js to make sure...

File: src/prepare-build.ts [hunk 2]
```diff
@@ -291,8 +293,19 @@ async function getPreviousVersionInfo(shortName: string, publishDate: string) {
 
 		const thisDate = thisURI.match(/[1-2][0-9]{7}/)![0];
```
Comment (deniak) [inline src/prepare-build.ts:294]: Not sure if it's related but I'm getting `SyntaxError: Unexpected token '!'`
```suggestion
		const thisDate = thisURI.match(/[1-2][0-9]{7}/)[0];
```

Comment (sidvishnoi) [reply src/prepare-build.ts:294]: It's TypeScript; needs to be compiled (`npm run build` or `yarn build`).

Comment (deniak) [reply src/prepare-build.ts:294]: Sorry, I was comparing the code with spec-generator and didn't pay attention to that.

**Pull request closed without merging**